Dockerfile

```diff
@@ -60,7 +60,13 @@ RUN set -ex && apk --update --no-cache add \
     bash \
     libstdc++ \
     libc6-compat \
-    ca-certificates
+    ca-certificates \
+    nodejs \
+    nodejs-npm
+
+# Add TypeScript support
+
+RUN npm i -g ts-protoc-gen@0.10.0
```

Comment:
I wondered if it's OK to install `nodejs` and `ts-protoc-gen` here, or if they should be moved to the `build` stage and copied following the examples below?

Reply:
i'm fine with it here; i think it needs to be here to make running the `node` command easier in the runtime container.